common/core/gpudevice.cpp

```diff
@@ -71,6 +71,7 @@ class GpuDevice::DisplayManager : public HWCThread {
       std::shared_ptr<DisplayHotPlugEventCallback> callback);
 
  protected:
+  void HandleWait() override;
```

Comment:
Can you add details like what was the problem and in Tests, what should be fixed now ? in u commit message